actors/builtin/power/power_state.go

```diff
@@ -98,15 +97,15 @@ func (st *State) minerNominalPowerMeetsConsensusMinimum(s adt.Store, minerPower
 		if err != nil {
 			return err
 		}
-		minerSizes = append(minerSizes, nominalPower)
+		if maxPower.LessThan(nominalPower) {
+			maxPower = nominalPower
+		}
 		return nil
 	}); err != nil {
 		return false, errors.Wrap(err, "failed to iterate power table")
 	}
 
-	// get size of MIN_MINER_SIZE_TARGth largest miner
-	sort.Slice(minerSizes, func(i, j int) bool { return i > j })
-	return minerPower.GreaterThanEqual(minerSizes[ConsensusMinerMinMiners-1]), nil
+	return minerPower.GreaterThanEqual(maxPower), nil
```

Comment:
This appears to change the semantics. The (poorly documented) goal is to return true for the top `ConsensusMinerMinMiners` miners.